Build instructions do work "out of the box"

- Build instructions do not work "out of the box" on Windows 11 or Mac
- Not all requirements are captured and the user must manually install extra packages
- rebuild_resource.sh script works on Mac but not Windows
- BUILD_WINDOWS.md doesn't capture exactly what console tool should be used (e.g. Git BASH, command prompt, Power Shell); instructions/scripts do not work correctly in all these tools
-

My results (Windows 11 Home):
- Git CMD console: 'sh is not recognized as internar or external command'. This is a fresh install of Git, so I must have to add sh executable to the path.
- 
- Git BASH: cannot find pyside executables in scripts/enlighten_py311 virtual environment.  The code that checks for the executable should be in double quotes to prevent globbing/word splitting (see [this](https://github.com/koalaman/shellcheck/wiki/SC2086))

Changing:
`(109) if ! [ -f $RCC ]`

to  
`(109) if ! [ -f "$RCC" ]`

Resolves this issue.  I think it's probably triggered by differences in our paths since my files are under a user directory that has a space in it.

--- a/scripts/rebuild_resources.sh
+++ b/scripts/rebuild_resources.sh
@@ -164,7 +164,7 @@ then
         DEST=${ROOT}_rc.py
 
         echo -n "converting $FILE"
-        $RCC $FILE -o $DEST $RCC_OPTS
+        "$RCC" $FILE -o $DEST $RCC_OPTS
         echo "...done"
     done
     echo
@@ -182,9 +182,9 @@ do
         continue
     fi
 
-echo -n "converting $FILE"
-$UIC_PRE $UIC $FILE -o $DEST $UIC_OPTS && convertToPy3 $DEST
-echo "...done"
+    echo -n "converting $FILE"
+    $UIC_PRE "$UIC" $FILE -o $DEST $UIC_OPTS && convertToPy3 $DEST
+    echo "...done"
 done
 
 # make uic_qrc a valid Python module

--- a/scripts/rebuild_resources.sh
+++ b/scripts/rebuild_resources.sh
@@ -113,12 +113,12 @@ echo "2to3        = $TWO_TO_THREE"
 echo
 
 FAILED=false
-if ! [ -f $RCC ]
+if ! [ -f "$RCC" ]
 then
     echo "Error: can't locate RCC: $RCC"
     FAILED=true
 fi
-if ! [ -f $UIC ]
+if ! [ -f "$UIC" ]
 then
     echo "Error: can't locate UIC: $UIC"
     FAILED=true

